packages/contentstack-migrate-rte/README.md

```diff
@@ -16,7 +16,7 @@ $ npm install -g @contentstack/cli-cm-migrate-rte
 $ csdx COMMAND
 running command...
 $ csdx (--version)
-@contentstack/cli-cm-migrate-rte/1.5.2 darwin-x64 node-v22.17.0
+@contentstack/cli-cm-migrate-rte/1.5.2 darwin-arm64 node-v22.14.0
```

Comment:
If there is no change in the plugin, you don't need to push the README files too.